pkg/operator/ceph/config/centralized.go

```diff
@@ -0,0 +1,48 @@
+package config
+
+import "github.com/coreos/pkg/capnslog"
+
+// DefaultCentralizedConfigs returns a map
+func DefaultCentralizedConfigs(rookLogLevel capnslog.LogLevel) *Config {
```

Comment:
Although this PR doesn't support the mon kv store for configs, only a config file (I wanted to support luminous+ and do the simplest amount of work), you can see that this is how default configs can be specified using the new config setup and also the configs intended to live in the kv store eventually.